tokio/src/io/async_fd.rs

```diff
@@ -519,6 +519,7 @@ impl<'a, Inner: AsRawFd> AsyncFdReadyGuard<'a, Inner> {
     /// create this `AsyncFdReadyGuard`.
     ///
     /// [`WouldBlock`]: std::io::ErrorKind::WouldBlock
+    #[doc(alias = "with_io")]
```

Comment:
```suggestion
    #[cfg_attr(docsrs, doc(alias = "with_io"))]
```

Reply:
I don't see the benefit of disabling it for local `cargo doc`.

Reply:
`doc(alias)` is not stable in tokio's MSRV, so we need to use `doc(alias)` only when cfg for documentation is enabled. (Sorry for forgot to explain why these changes are needed!)
https://github.com/tokio-rs/tokio/runs/1897938802?check_suite_focus=true#step:4:93

Reply:
I see. Thanks